Gemfile.lock

```diff
@@ -73,10 +73,10 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: 0832461353f871b688b0c231ecf9bdbfb1f9145a
+  revision: 7176cb057f11f66f16889005a1873a084bad188c
   branch: master
   specs:
-    vets_json_schema (22.0.3)
+    vets_json_schema (22.1.1)
```

Comment:
Yes, because vets-website has gotten updated to `22.1.0` since then (https://github.com/department-of-veterans-affairs/vets-website/blob/0ee8639c13d4da5c4520e70c2e79bb75236407c5/yarn.lock#L20599), so it seemed best to keep them in sync